### PR TITLE
Add instruction about languages and promotion

### LINKED ISSUE #8: Ajouter des conseils pour mettre en valeur la communauté HACF dans les dépots hebergés
Donner des conseils dans la partie `CONTRIBUTING.md` pour ajouter les badges vers le Forum, Le Discord HACF et l'awesome list.

### PR DESCRIPTION
**Pull request by oncleben31**
- fix: #8 
- Add good practices for languages to be used in our organization.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,3 +52,28 @@ hébergée dans l'organisation est là pour regrouper toutes les ressources inte
 pour les francophones. Elle doit être vivante et peut être améliorée par tous.
 N'hésitez pas à contirbuer. Des instructions détaillées sont disponible dans le
 dépôt.
+
+## Promotion de la communauté
+
+N'oublions pas de faire la promotion de la communauté dans les dépôts que nous maintenons.
+Nous recommandons d'utilser quand cela est pertinent les badges suivants:
```
Comment (Quentame) [inline CONTRIBUTING.md]:
```suggestion
Nous recommandons d'utiliser quand cela est pertinent les badges suivants:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -52,3 +52,28 @@ hébergée dans l'organisation est là pour regrouper toutes les ressources inte
...
+
+* [![Discord francophone][discord-shield-fr]][discord-fr]: pour la promotion du Discord
+* [![Forum francophone][forum-shield-fr]][forum-fr]: pour la promotion du Forum
+* [![Awesome francophone][awesome-shield]][awesome-fr]: pour la promotion de l'awesome list. Surtout si il y a une entrée dans la liste en lien avec le dépôt
```
Comment (Quentame) [inline CONTRIBUTING.md]:
```suggestion
* [![Awesome francophone][awesome-shield]][awesome-fr]: pour la promotion de l'awesome list. Surtout s'il y a une entrée dans la liste en lien avec un dépôt HACF
```

File: CONTRIBUTING_EN.md [hunk 3]
```diff
@@ -51,3 +51,30 @@ resources linked to Home Assistant. The list needs to be regularly updated and i
 
 Please contribute. Detailed instructions are
 available in the repository.
+
+## Community promotion
+
+We should not forget to promote the community in the repository we maintain.
```
Comment (Quentame) [inline CONTRIBUTING_EN.md]:
```suggestion
We should not forget to promote the community in repositories we maintain.
```

File: CONTRIBUTING_EN.md [hunk 4]
```diff
@@ -51,3 +51,30 @@ resources linked to Home Assistant. The list needs to be regularly updated and i
...
+We recommend to use the following badges when applicable.
+
+- [![Discord francophone][discord-shield-fr]][discord-fr]: to promote the Discord
+- [![Forum francophone][forum-shield-fr]][forum-fr]: to promote the Forum
+- [![Awesome francophone][awesome-shield]][awesome-fr]: to promote the awesome list. Mostly if there is an entry in the list related to the repository.
```
Comment (Quentame) [inline CONTRIBUTING_EN.md]:
```suggestion
- [![Awesome francophone][awesome-shield]][awesome-fr]: to promote the awesome list. Mostly if there is an entry in the list related to an HACF repository.
```

File: CONTRIBUTING_EN.md [hunk 5]
```diff
@@ -51,3 +51,30 @@ resources linked to Home Assistant. The list needs to be regularly updated and i
...
+
+## Language used
+
+English is standard on GitHub.
+We recommand to follow this best practice as much as possible.
+
+On the other hand, we have to welcome and be friendly with the French speaking contributors.
+
+It is allowed to prefer French language on repository related to services or products targeting only francophones
```
Comment (Quentame) [inline CONTRIBUTING_EN.md]:
```suggestion
It is allowed to prefer French language on repositories related to services or products targeting only francophones
```

**Review by Quentame — CHANGES_REQUESTED** [review]
Almost

**Comment by oncleben31** [comment]
Merci pour la revue. Remarques prises en compte.

File: CONTRIBUTING_EN.md [hunk 6]
```diff
@@ -77,4 +77,4 @@ In this case, we have to make this choice visible.
 [forum-shield-fr]: https://img.shields.io/discourse/topics?label=Forum%20francophone%20HACF&logo=discourse&server=https%3A%2F%2Fforum.hacf.fr%2F
 [forum-fr]: https://forum.hacf.fr/
 [discord-shield-fr]: https://img.shields.io/discord/706096417000652840?label=Discord%20francophone%20HACF&logo=discord
-[discord-fr]: https://discord.com/invite/PaZFEjX
\ No newline at end of file
+[discord-fr]: https://discord.com/invite/PaZFEjX
```
Comment (Quentame) [inline CONTRIBUTING_EN.md:80]: This should not change EOL issue ?

**Review by Quentame — APPROVED** [review]

**Pull request merged**